caffe2/CMakeLists.txt

```diff
@@ -1,41 +1,70 @@
-# Excludes
+# ---[ Excludes
 set(Caffe2_DIR_EXCLUDES
-        "/binaries"
-        "/contrib"
-        "/distributed"
-        "/mpi"
-        "/image"
```

Comment:
some of the files in here have openMP requirements, breaking the build if you don't exclude them by default. They should only be included if openmp is available.